app/views/events/_applicants_overview.html.erb

```diff
@@ -13,7 +13,7 @@
         <%= sort_caret(Profile.human_attribute_name(:gender), :gender) %>
       </th>
       <th>
-        <%= sort_caret(Profile.human_attribute_name(:age), :age) %>
+        <%= sort_caret(( t('.age') + " <br> " + t('.when_event_starts')), :applicant_age_when_event_starts ) %>
```

Comment:
It doesn't break the line if you don't do it maually. Thing is, that it fits into the line, but that makes the columns besides that one smaller which leads to a linebreak in the status-update-toggle-fields. You know what I'm talking about?